account_check_printing_report_base/__manifest__.py

```diff
@@ -16,7 +16,7 @@
     "category": "Generic Modules/Accounting",
     "website": "https://github.com/OCA/account-payment",
     "depends": ["account_check_printing", "report_wkhtmltopdf_param"],
-    "external_dependencies": {"python": ["num2words"]},
+    "external_dependencies": {"python": ["num2words==0.5.6"]},
```

Comment:
Please remove this

Reply:
Should I change it to `num2words<=0.5.10`, which the current one and the one we are effectively supporting here?